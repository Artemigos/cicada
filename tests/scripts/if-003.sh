echo hi
counter=17
if echo foo | grep -q oo
    while echo "$counter" | grep -iq "^1.$"
        echo "counter = $counter"
        counter=$(expr $counter + 1)
    done
fi

if echo foo | grep -q bar
    # will not enter this if
    while echo "$counter" | grep -iq "^2.$"
        echo "counter = $counter"
        counter=$(expr $counter + 1)
    done
fi

if echo foo | grep -q oo
    if echo bar | grep -q oo
        echo found oo
    else
        while echo "$counter" | grep -iq "^2[0-2]$"
            echo "counter = $counter"
            counter=$(expr $counter + 1)
        done
    fi
fi

if echo foo | grep -q oo
    if echo bar | grep -q bar
        echo found bar
    else
        while echo "$counter" | grep -iq "^2.$"
            echo "counter = $counter"
            counter=$(expr $counter + 1)
        done
    fi
fi

echo bye
