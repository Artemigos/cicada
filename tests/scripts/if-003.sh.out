hi
counter = 17
counter = 18
counter = 19
counter = 20
counter = 21
counter = 22
found bar
bye
